packages/sql-mapper/mapper.d.ts

```diff
@@ -18,7 +18,7 @@ export interface Database {
   /**
    * An option that is true if a MySQL database is used.
    */
-  isMySQL?: boolean,
```

Comment:
Why CI didn't catch this?

Reply:
I suppose because the wasn't a test for it.

Reply:
it was defined `isMySQL` in `mapper.js` , fixed in 47da73